xml_converter/integration_tests/test_cases/texture_invalid/testcase.yaml

```diff
@@ -0,0 +1,7 @@
+input_paths: 
+    "pack": "xml"
+expected_stdout: |
+    Warning: File path test_cases/texture_invalid/input/pack/texture_01.png not found.
+    Warning: File path test_cases/texture_invalid/input/pack/texture_01.png not found.
```

Comment:
Why is this printed twice?

Reply:
Double errors are expected as there is a check when writing the proto and a check when writing the xml